src/dbnode/config/m3dbnode-local-etcd.yml

```diff
@@ -3,6 +3,44 @@ coordinator:
     type: "config"
     value: "0.0.0.0:7201"
 
+  clusters:
```

Comment:
This is not necessary for the embedded m3dbnode case, it will auto create a client for the local node.  This is adding a pretty large overhead of config that's not necessary.